dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -19,45 +19,41 @@
-JAVAEESEC_ERROR_NO_VALIDATION.explanation=There was no IdentityStore found to validate the user, therefore the user cannot be authenticated.
-JAVAEESEC_ERROR_NO_VALIDATION.useraction=Make sure that at least one IdentityStore supports validating the user.
+JAVAEESEC_ERROR_NO_VALIDATION=CWWKS1911E: No IdentityStore object supports validation of the user. Make sure that at least one IdentityStore object supports user validation.
+JAVAEESEC_ERROR_NO_VALIDATION.explanation=No IdentityStore object was found to validate the user. Therefore, the user cannot be authenticated.
+JAVAEESEC_ERROR_NO_VALIDATION.useraction=Make sure that at least one IdentityStore object supports validation of the user.
 
-JAVAEESEC_ERROR_NO_HAM=CWWKS1912E: The HttpAuthenticationMechanism for the {0} module in the {1} application could not be created.
-JAVAEESEC_ERROR_NO_HAM.explanation=The HttpAuthenticationMechanism cannot be created.
-JAVAEESEC_ERROR_NO_HAM.useraction=Investigate any error messages from the HttpAuthenticationMechanism.
+JAVAEESEC_ERROR_NO_HAM=CWWKS1912E: The HttpAuthenticationMechanism object for the {0} module in the {1} application could not be created.
+JAVAEESEC_ERROR_NO_HAM.explanation=The HttpAuthenticationMechanism object cannot be created. <= Give additional information to explain why it wasn't found, maybe some examples of what the problem could be. <<< There are various cause why thie error is reported. A user need to review errors in a message.log file.
+JAVAEESEC_ERROR_NO_HAM.useraction=Investigate any error messages from the HttpAuthenticationMechanism object.
 
 #Note to translator, do not translate EL.
-JAVAEESEC_WARNING_IDSTORE_CONFIG=CWWKS1916W: There was an error evaluating the Expression Language (EL) expression for the ''{0}'' attribute of the identity store annotation. Ensure that the EL expression and the result are valid and ensure any referenced beans that are used in the expression are resolvable. The default attribute value ''{1}'' will be used.
-JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=The identity store attribute Expression Language expression cannot be resolved to a valid value.
-JAVAEESEC_WARNING_IDSTORE_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. If you use EL expressions, ensure that they are valid and that any referenced beans that are used in the expression are resolvable.
+JAVAEESEC_WARNING_IDSTORE_CONFIG=CWWKS1916W: The Expression Language (EL) expression for the ''{0}'' attribute of the identity store annotation cannot be resolved to a valid value. Ensure that the EL expression and the result are valid and ensure that any referenced beans that are used in the expression are resolvable. The default attribute value of ''{1}'' is used instead.
+JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=An error occurred when evaluating the EL expression for the specified attribute. <= Give additional information to explain why it wasn't found, maybe some examples of what the problem could be.
```

Comment:
Remove the "<= Give additonal information....".
Explain what a valid configuration value is or give examples of the most common  values. If you are stuck on what to put here, explain to me what the valid values are and give me some common examples. We can figure out the wording together. Also, we want to start the explanation with something other than "An error occurred..".

Reply:
An example was added.